src/Text/Pretty/Simple/Internal/Color.hs

```diff
@@ -25,262 +27,110 @@ module Text.Pretty.Simple.Internal.Color
-
--- | Default colors for 'colorRainbowParens' for light backgrounds.
-defaultColorRainbowParensLightBg :: [AnsiStyle]
-defaultColorRainbowParensLightBg =
-  [ colorVividMagentaBold
-  , colorVividCyanBold
-  , colorDullMagenta
-  , colorDullCyan
-  , colorDullMagentaBold
-  , colorDullCyanBold
-  , colorVividMagenta
-  , colorVividCyan
-  ]
-
------------------------
--- Vivid Bold Colors --
------------------------
-
-colorVividBlackBold :: AnsiStyle
-colorVividBlackBold = colorBold `mappend` colorVividBlack
-
-colorVividBlueBold :: AnsiStyle
-colorVividBlueBold = colorBold `mappend` colorVividBlue
-
-colorVividCyanBold :: AnsiStyle
-colorVividCyanBold = colorBold `mappend` colorVividCyan
-
-colorVividGreenBold :: AnsiStyle
-colorVividGreenBold = colorBold `mappend` colorVividGreen
-
-colorVividMagentaBold :: AnsiStyle
-colorVividMagentaBold = colorBold `mappend` colorVividMagenta
-
-colorVividRedBold :: AnsiStyle
-colorVividRedBold = colorBold `mappend` colorVividRed
-
-colorVividWhiteBold :: AnsiStyle
-colorVividWhiteBold = colorBold `mappend` colorVividWhite
-
-colorVividYellowBold :: AnsiStyle
-colorVividYellowBold = colorBold `mappend` colorVividYellow
-
------------------------
--- Dull Bold Colors --
------------------------
-
-colorDullBlackBold :: AnsiStyle
-colorDullBlackBold = colorBold `mappend` colorDullBlack
-
-colorDullBlueBold :: AnsiStyle
-colorDullBlueBold = colorBold `mappend` colorDullBlue
-
-colorDullCyanBold :: AnsiStyle
-colorDullCyanBold = colorBold `mappend` colorDullCyan
-
-colorDullGreenBold :: AnsiStyle
-colorDullGreenBold = colorBold `mappend` colorDullGreen
-
-colorDullMagentaBold :: AnsiStyle
-colorDullMagentaBold = colorBold `mappend` colorDullMagenta
-
-colorDullRedBold :: AnsiStyle
-colorDullRedBold = colorBold `mappend` colorDullRed
-
-colorDullWhiteBold :: AnsiStyle
-colorDullWhiteBold = colorBold `mappend` colorDullWhite
-
-colorDullYellowBold :: AnsiStyle
-colorDullYellowBold = colorBold `mappend` colorDullYellow
-
-------------------
--- Vivid Colors --
-------------------
-
-colorVividBlack :: AnsiStyle
-colorVividBlack = color Black
-
-colorVividBlue :: AnsiStyle
-colorVividBlue = color Blue
-
-colorVividCyan :: AnsiStyle
-colorVividCyan = color Cyan
-
-colorVividGreen :: AnsiStyle
-colorVividGreen = color Green
-
-colorVividMagenta :: AnsiStyle
-colorVividMagenta = color Magenta
-
-colorVividRed :: AnsiStyle
-colorVividRed = color Red
-
-colorVividWhite :: AnsiStyle
-colorVividWhite = color White
-
-colorVividYellow :: AnsiStyle
-colorVividYellow = color Yellow
-
-------------------
--- Dull Colors --
-------------------
-
-colorDullBlack :: AnsiStyle
-colorDullBlack = colorDull Black
-
-colorDullBlue :: AnsiStyle
-colorDullBlue = colorDull Blue
-
-colorDullCyan :: AnsiStyle
-colorDullCyan = colorDull Cyan
-
-colorDullGreen :: AnsiStyle
-colorDullGreen = colorDull Green
-
-colorDullMagenta :: AnsiStyle
-colorDullMagenta = colorDull Magenta
-
-colorDullRed :: AnsiStyle
-colorDullRed = colorDull Red
```

Comment:
I may have gone overboard in removing things from this module. But with this more user-friendly colour type (previously it was just `Builder`), it seems a bit pointless to export all these combinations.